Libraries/LibWeb/Page/EventHandler.cpp

```diff
@@ -1283,12 +1283,32 @@ EventResult EventHandler::handle_keydown(UIEvents::KeyCode key, u32 modifiers, u
     auto* target = document->active_input_events_target();
     if (target) {
         if (key == UIEvents::KeyCode::Key_Backspace) {
+            if ((modifiers & UIEvents::KeyCode::Key_LeftControl) != 0) {
```

Comment:
nit: virtually just noting that probably depends on the taste and might be os-dependent, but you probably should check whether `modifiers` exactly match given combination, e.g. left control, not whether it contains it?

(it might be just more key combinations should be introduced here, in proper order, as each action is marked as handled, so current resolution is just okay.)